lk2nd/device/dts/msm8952/msm8917-huawei-agassi.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+
+/*
+ * This device is quirky. It excluded from generic build.
+ * To run lk2nd on this device you need to flash custom bootloader and unlock it. Read more https://wiki.postmarketos.org/wiki/Huawei_Mediapad_T3_10_(huawei-agassi)#Bootloader_unlocking.
+ *
+ * Compile lk2nd with the following flag to run on this device
+ * LK2ND_ADTBS="msm8917-huawei-agassi.dtb"
+ */
```

Comment:
I'd appreciate the comment to be folded at 80-100 where possible so it's a bit easier to read